ruby/cross_service_examples/item_tracker/README.md

```diff
@@ -0,0 +1,102 @@
+
+## Create the resources
+Set up the AWS resources needed to run this example as follows:
+1. Create AWS infrastructure using [instructions in the Aurora Serverless App README.md](../../../resources/cdk/aurora_serverless_app/README.md).
+   * NOTE: Store the following output values in the `env/config.yml` file: `CLUSTER_ARN`, `SECRET_ARN`, and `DATABASE`.
+2. Populate your database with a table and data using the following commands:
+     ```bash
+     cd ruby/cross_service_examples/item-tracker/helpers
+     ruby create_table.rb # checks for database and creates a new table
+     ruby populate_data.rb # creates a table with 10 records
+     ```
+3. Register your email address with Amazon Simple Email Service (SES) using the instructions in [Creating and verifying identities in Amazon SES](https://docs.aws.amazon.com/ses/latest/dg/creating-identities.html)
+   * NOTE: Use this email for the `sender_email` and `recipient_email` fields in your `env/config.yml` file.
+
+## Build the code
+To view this example in its entirety, you must:
+1. Start the frontend React application using the following commands:
+    ```bash
+     cd resources/clients/elwing # located within Resources sub-directory
+     npm install # first time only
+     npm start # launches a browser session on localhost:3000 
+    ```
+2. Open a second tab in your terminal and run the following command:
+    ```bash
+    cd ruby/cross_service_examples/item-tracker
+    ruby app.rb # starts REST API listening on port 8080
+    ```
+Now, visit http://localhost:3000/item_tracker to view your working application.
+
+## Test the code
+Test this application manually via the frontend or automatically via the RSpec tests.
+
+### Manual validation (via frontend React app)
+Once the app is started, manually validate the following behaviors on the frontend:
+* Filter items by All, Archived, and Active (not archived).
+* Archive an item by clicking the Archive button.
+* Add an item by submitting the form revealed by clicking the Add Item button.
+* Send a report of items by entering a verified email address and clicking Submit.
+
+As you interact with the React app, logs will appear in the command line tab where you started the REST API.
+
+### Automated validation (via RSpec tests)
+Validate the internal logic within this example by running the following commands:
+```bash
+cd spec
+rspec db_wrapper_spec.rb
+```
+
+## Delete the resources
+
+To avoid charges, delete all the resources that you created for this tutorial. Follow the instructions in the Destroying resources section of the README for the Aurora Serverless sample application.
+
+## Additional resources
+* [Service Developer Guide](https://docs.aws.amazon.com/sdk-for-ruby/v3/developer-guide/welcome.html)
+* [Service API Reference](https://docs.aws.amazon.com/sdk-for-ruby/v3/api/)
+* [SDK API reference guide](https://aws.amazon.com/developer/language/ruby/)
```

Comment:
Use specific titles instead of generic ones. Also, these seem a little mixed up. They are meant to be:
* [Aurora User Guide](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/CHAP_AuroraOverview.html)
* [Amazon SES Developer Guide](https://docs.aws.amazon.com/ses/latest/dg/Welcome.html)
* [SDK for Ruby Developer Guide](https://docs.aws.amazon.com/sdk-for-ruby/v3/developer-guide/welcome.html)
* [SDK for Ruby API Reference](https://docs.aws.amazon.com/sdk-for-ruby/v3/api/)